app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,36 @@
+class OrdersController < ApplicationController
+  before_action :logged_in_user 
+  
+  def index
+    set_orders
+  end
+
+  def destroy
+    Order.find(params[:id]).destroy
+    flash[:success] = "Order Canceled"
+    redirect_to orders_url
+  end
+
+  def toggle
+    @order = Order.find(params[:order_id])
+    all_orders
```

Comment:
I think it's clear if you just say `@orders = Order.all` instead of that method. Or rename it to `set_all_orders`